test/distributed/fsdp/test_fsdp_meta.py

```diff
@@ -225,7 +235,9 @@ def _test_nested_model_with_meta_device(
     ):
         if auto_wrap:
             module = meta_module_fn()
-            is_meta = next(module.parameters()).is_meta
+            is_meta = (
+                next(module.parameters()).is_meta or next(module.buffers()).is_meta
```

Comment:
thanks for adding the buffers! would a test with only buffers being meta have picked this issue up? if so, should we add it?

Reply:
Yes, I had to add this since I added a buffer-only module.